src/main/java/frc/robot/commands/ShooterBaseCommand.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.commands;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.shooter.ShooterBase;
+import java.util.Map;
+import java.util.function.BooleanSupplier;
+
+public class ShooterBaseCommand extends Command {
+  ShooterBase shooter;
+  BooleanSupplier enable;
+  ShuffleboardTab tab;
+  GenericEntry voltsEntry;
+
+  public ShooterBaseCommand(ShooterBase shooter, BooleanSupplier enable) {
+    this.shooter = shooter;
+    this.enable = enable;
+
+    // create shooter tab on ShuffleBoard
+    tab = Shuffleboard.getTab("Shooter");
+    // Create volt entry under Shooter tab as a number sider with min = -1 and max = 1
+    voltsEntry =
+        tab.add("Volts", 0)
+            .withWidget(BuiltInWidgets.kNumberSlider)
+            .withProperties(Map.of("min", -12, "max", 12))
+            .getEntry();
+
+    // Sets default value to 0.0
+    voltsEntry.setValue(0.0);
+
+    addRequirements(shooter);
+  }
+
+  @Override
+  public void initialize() {
+    // disable shooter when initialize
+    shooter.disable();
+  }
```

Comment:
Depending on how this command is used, you may not want to disable() the shooter here. The initialize() method is called whenever the command is _scheduled_, so if this command is called after other commands that may already have turned on the shooter (e.g maybe at a slower velocity), you wouldn't want to disable() the shooter only to turn it back on in the execute() command (if enable=true).   This could cause the shooter to hiccup when this command is scheduled.  You might want to just leave the initialize() command out.

Reply:
I see, I will remove shooter.disable()